src/commands/markov.rs

```diff
@@ -54,7 +57,61 @@ command!(generate_user(ctx, message, args) {
         if let Some(markov) = get_uid(arg)
                 .map_or(None,
                         |x| usermap.get(&x)) {
-            output_markov(&markov, &message, length);
+            output_markov(&markov, &message, length, None);
+        }
+    }
+});
+
+
+command!(generate_from_word(ctx, message, args) {
+    let mut data = ctx.data.lock().unwrap();
+    let mut markov = data.get_mut::<Markov>().unwrap();
+    let mut word: Option<&str> = None;
```

Comment:
No I mean like, why not set it's value either within a function or within a `{}` scope
and then set it to a non-`mut` binding then?